### PR TITLE
[dong] update cobar page #711

### PR DESCRIPTION
**Pull request by phongvu2812**
# Ticket
 #711

# General
- [ ] No unnecessary files, debug logs, or commented code included.
- [ ] PR has at least 1 peer approval before senior review.
- [ ] I fully understand the code I submitted.
- [ ] No hardcoded configuration or secrets.
- [ ] **Environment Variables** changes? (Leave unchecked if N/A)

# Design & Solution Clarity
- [ ] A design diagram (sequence / flow) is attached or linked.
- [ ] The implementation matches the design.
- [ ] All major flows and decision points are represented in the diagram.
- [ ] Error handling and edge cases are clearly defined in the design.

Design link (diagram / doc):
`<paste link here>`

# Testing
- [ ] Unit tests cover main logic and important edge cases.

> Note: Complex logic must not rely on manual testing only.

# UI evidence (For frontend only)
`<paste screenshots here>`

# Peer Review Checklist
- [ ] I understand the overall solution and flow.
- [ ] The code follows the provided design diagram.
- [ ] No obvious logic or security issues found.
- [ ] Tests are sufficient for the scope of this change.

### CODE REVIEW

File: frontend/modules/cobar/components/shared/card.tsx [hunk 1]
```diff
@@ -21,13 +22,29 @@ export default function Card({
...
         className
       )}
+      onClick={onClick}
+      role={isClickable ? 'button' : undefined}
+      tabIndex={isClickable ? 0 : undefined}
+      onKeyDown={
+        isClickable
+          ? (e) => {
+              if (e.key === 'Enter' || e.key === ' ') {
+                onClick();
```
Comment (vietdangquoc) [inline frontend/modules/cobar/components/shared/card.tsx]: - thêm      e.preventDefault();
- using e.Code

**Review by vietdangquoc — APPROVED** [review]

**Review by hoangdohuy — APPROVED** [review]
LGTM
